core/src/main/java/org/apache/iceberg/inmemory/InMemoryOutputFile.java

```diff
@@ -30,7 +33,7 @@ public class InMemoryOutputFile implements OutputFile {
 
   private boolean exists = false;
   private ByteArrayOutputStream contents;
-  private InMemoryFileIO parentFileIO;
+  private final InMemoryFileIO parentFileIO;
```

Comment:
nit: can we place final variables over non final ones

Reply:
sure. done